expressEngine/user/route.js

```diff
@@ -11,5 +11,5 @@ const router = express.Router();
 
 router.post('/', parser(UserController, 'create'));
 router.get('/', parser(UserController, 'get'));
-router.delete('/:id', parser(UserController, 'delete'));
+router.delete('/:id', parser(UserController, 'login'));
```

Comment:
why are you using delete method for login?

Reply:
change login to delete done